twake/backend/node/src/services/channels/index.ts

```diff
@@ -28,6 +33,19 @@ export default class ChannelService extends TwakeService<ChannelServiceAPI> {
       next();
     });
 
+    phpnode.register((internalServer, _, next) => {
+      internalServer.route({
+        method: "GET",
+        url: "/companies/:company_id/workspaces/:workspace_id/channels/:id/members/:member_id",
```

Comment:
This route already exists

Reply:
No here this is the internal route (without /service/channels/v1/ prefix)
The final route is "/private/companies/etc"

I need the phpnode service to provide a new set of routes prefixed /private for internal communication (and so not open to the world)